custom_components/midea_ac_lan/__init__.py

```diff
@@ -188,7 +188,7 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     model: str = config_entry.data[CONF_MODEL]
     subtype = config_entry.data.get(CONF_SUBTYPE, 0)
     protocol: int = config_entry.data[CONF_PROTOCOL]
-    customize: str = config_entry.options[CONF_CUSTOMIZE]
+    customize: str = config_entry.options.get(CONF_CUSTOMIZE, "")
```

Comment:
We should do a separate PR to fix bugs

Reply:
Agree

Reply:
done, via https://github.com/wuwentao/midea_ac_lan/pull/115